engine/plugins/server/package.json

```diff
@@ -1,22 +1,23 @@
 {
-  "name": "server",
-  "description": "Server for establishing socket.io connections to clients",
-  "version": "1.0.0",
-  "main": "server.js",
-  "private": true,
-  "plugin": {
-    "consumes": [
-      "bus"
-    ],
-    "provides": [
-      "app",
-      "server"
-    ]
-  },
-  "dependencies": {
-    "express": "^4.17.1",
-    "node-fetch": "^2.6.1",
-    "socket.io": "^2.3.0",
-    "uniqid": "^5.2.0"
-  }
+    "name": "server",
```

Comment:
Why the re-spacing... it's not related to the PR as such? Will accept it but try to split reformatting code an actually code changes in diffrent PR's. It makes review easier :-) and time is not wasted on looking for changes.

Reply:
I added client as requirement. And default is 4 spaces, so I changed the alignment as well.

Reply:
I did not spot that change... :-)